content/en/blog/2024/new-otel-features-envoy-istio/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: 'Observability on the Edge: Enhancing OTel support in Envoy and Istio'
+linkTitle: New OTel features in Envoy and Istio
+date: 2024-05-28
+author: '[Joao Grassi](https://github.com/joaopgrassi) (Dynatrace)'
+issue: 4534
+sig: OpenTelemetry Specification
+---
+
+In the dynamic world of cloud-native and distributed applications, managing
+microservices effectively is critical. Kubernetes has become the de facto
+standard for container orchestration, enabling seamless deployment, scaling, and
+management of containerized applications.
+
+The distributed nature of such systems, however, adds a layer of complexity in
+the form of networking for in-cluster communication. Two well-known projects,
+Envoy and Istio, have emerged as the foundation for the smooth management and
+operation of such complex environments.
+
+Together, these technologies empower organizations to build scalable, resilient,
+and secure distributed systems.
+
+[Istio](https://istio.io/) is a service mesh, that orchestrates communication
+between microservices, providing features such as traffic management, security
+and, of course observability. Istio uses the Envoy proxy as its data plane.
+[Envoy](https://www.envoyproxy.io/) is a high-performance proxy, designed for
+single applications/services as well as a communication bus and "universal data
+plane" for service meshes.
+
+[Envoy](https://www.cncf.io/projects/envoy/) and
+[Istio](https://www.cncf.io/projects/istio/) projects are open source and part
+of the Cloud Native Computing Foundation.
+
+## Observability in Envoy and Istio
+
+The Envoy proxy deployed by the Istio service mesh is the perfect candidate to
+ensure incoming and outgoing requests are properly traced. This approach
+provides distributed traces of the entire service mesh, giving an overview on
+the communication between services — even when the applications themselves are
+not instrumented.
+
+Envoy offers several
+[tracers](https://www.envoyproxy.io/docs/envoy/v1.29.4/api-v3/config/trace/trace)
```

Comment:
is there a reason why this is pinned to version 1.29.4?

Reply:
I thought about pinning to 1.29 as that is when the new features became available. I guess we could also link to `latest` but that can be misleading if somehow the link changes or whatever else. Not sure what is better 🤔

Reply:
That makes sense, let's keep it pinned then.